### ISSUE TITLE
Can we please get an updated version of this for TF2?

### ISSUE THREAD

**Issue by Lamm-a** [EdjeElectronics/TensorFlow-Object-Detection-API-Tutorial-Train-Multiple-Objects-Windows-10#483]
There are muntiple sections of this guide that are now out of date unfortunatly :( The work around of using the legacy `train.py` in 1.9 does not work in 2.0 and has no work around unfortunatly.

**Comment by JulianOrteil** [comment]
As posted multiple times here and elsewhere, the Tensorflow maintainers have not yet released a compatible version of the object detection api for TF 2.0.

Most notable reason: TF 1.x has a module called "contrib" that no longer exists in 2.0. Until they can work around that and any other barriers, you will need to use TF 1.x

**Comment by ashish735** [comment]
The Tensorflow maintainers have released the compatible version of the object detection api for TF 2.0. If we get an updated version of this for TF 2.0, It will be really great. Thank you

**Comment by wu-jin-liang** [comment]
Hi, everybody, I am new to tensorflow..
I use the tensorflow-1.13,  cudatoolkit , cudnn,  and models-r1.13.0.zip. I follow the tutorial , use the frozen inference graph from this Dropbox link https://www.dropbox.com/s/va9ob6wcucusse1/inference_graph.zip?dl=0
I can run the Object_detection_video.py demo successfully....
but then I try to train....
both train.py or model_main.py ,,,, 
It can init ok,,,,,,   but it always halt after start a while.

#######use model_main.py, halts at step =0.
INFO:tensorflow:loss = 3.8967483, step = 0
I0805 10:27:55.944391 19564 basic_session_run_hooks.py:262] loss = 3.8967483, step = 0

////////////////////////////////////////////////////////////////////////////
#######use train.py, halts at step >=90 , Saving checkpoint
I0805 10:50:53.543954 18848 learning.py:507] global step 97: loss = 2.5882 (1.244 sec/step)
INFO:tensorflow:global step 98: loss = 2.5304 (1.294 sec/step)
I0805 10:50:54.840963 18848 learning.py:507] global step 98: loss = 2.5304 (1.294 sec/step)
INFO:tensorflow:global step 99: loss = 2.0427 (0.907 sec/step)
I0805 10:50:55.750564 18848 learning.py:507] global step 99: loss = 2.0427 (0.907 sec/step)
INFO:tensorflow:Saving checkpoint to path training/model.ckpt
I0805 10:58:56.402550  2836 supervisor.py:1117] Saving checkpoint to path training/model.ckpt

***************************************************************************

after halt ,, the anaconda prompt no longer out put any information..
and gpu usage drop to 0%,,, but cpu usage is above 50%, ram  87%

I try to let it run for 10 hours after halt .  gpu is always 0%,,  cpu is always above 50% ,ram 87%...
but there is no result come out,,,,,
May I should wait for a long time? or it is something errors?
///////////////////////////////////////////////////////////////////////////////////////////////////
for my laptop,
Nitro AN515-52
Core i5-8300H @ 2.30GHz  8 GB RAM
Nvidia GeForce GTX 1050 ( 4 GB )

**Comment by ashish735** [comment]
Hi @wu-jin-liang there is some error. Your code should not halt at all.. Check if your RAM or VRAM is getting exhausted
